org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -407,6 +464,26 @@ package lang {
 		override method asSet() = self
 
 		override method anyOne() native
+
+		/**
+		 * Returns a new Set with the elements of both self and another collection.
+		 * @returns a Set
+		 */
+		 method union(another) = self + another
```

Comment:
Sorry, where is  + defined?

I see `union`, `difference` , etc documented as if they return a set, but it seems they are not.

```
method intersection(another) = self.filter({it => another.contains(it)}) 
```